lib/commands/element.js

```diff
@@ -81,19 +82,25 @@ commands.getElementScreenshot = async function (element) {
   return await this.uiautomator2.jwproxy.command(`/element/${element}/screenshot`, 'GET', {});
 };
 
-commands.tap = async function (element, x = 0, y = 0, count = 1) {
+commands.tap = async function (elementId = null, x = null, y = null, count = 1) {
+  const params = {};
+  const areCoordinatesDefined = util.hasValue(x) && util.hasValue(y);
+  if (!util.hasValue(elementId) && !areCoordinatesDefined) {
+    throw new Error(`Either element to tap or both absolute coordinates should be defined`);
+  }
+  if (areCoordinatesDefined) {
```

Comment:
Maybe it would be helpful to add log messages that ```elementId``` or ```x|y``` arguments will be ignored?